docs/reference/ml/df-analytics/apis/delete-trained-models.asciidoc

```diff
@@ -10,7 +10,7 @@
 Deletes an existing trained {infer} model that is currently not referenced by an
 ingest pipeline.
 
-experimental[]
+beta[]
```

Comment:
Why are some tags `beta::[]` but others `beta[]`?

Reply:
I think the latter was the pre-Asciidoctor format.  Since https://github.com/elastic/docs#using-the-beta-admonition recommends the former, I'll make them all the same.